spec/views/accounts/_work_details.html.erb_spec.rb

```diff
@@ -0,0 +1,550 @@
+        application.update!(referred_by_return_to_teaching_adviser: "yes")
+        render partial: "accounts/work_details", locals: { application: application }
+      end
+
+      it "shows the referred by return to teaching adviser row" do
+        expect(subject).to have_css(".govuk-summary-list__key", text: "Referred by return to teaching adviser")
+        expect(subject).to have_css(".govuk-summary-list__value")
+      end
+    end
+
+    context "when referred_by_return_to_teaching_adviser is nil" do
+      before do
+        application.update!(referred_by_return_to_teaching_adviser: nil)
+        render partial: "accounts/work_details", locals: { application: application }
+      end
+
+      it "does not show the referred by return to teaching adviser row" do
+        expect(subject).not_to have_css(".govuk-summary-list__key", text: "Referred by return to teaching adviser")
+      end
+    end
+
+    context "when referred_by_return_to_teaching_adviser is blank" do
+      before do
+        application.update!(referred_by_return_to_teaching_adviser: "")
+        render partial: "accounts/work_details", locals: { application: application }
+      end
+
+      it "does not show the referred by return to teaching adviser row" do
+        expect(subject).not_to have_css(".govuk-summary-list__key", text: "Referred by return to teaching adviser")
+      end
+    end
+  end
+
+  describe "work setting" do
+    context "when work_setting is present" do
+      before do
+        application.update!(work_setting: "school")
+        render partial: "accounts/work_details", locals: { application: application }
+      end
+
+      it "shows the work setting row" do
+        expect(subject).to have_css(".govuk-summary-list__key", text: "Work setting")
+        expect(subject).to have_css(".govuk-summary-list__value")
+      end
+    end
+
+    context "when work_setting is nil" do
+      before do
+        application.update!(work_setting: nil)
+        render partial: "accounts/work_details", locals: { application: application }
+      end
+
+      it "does not show the work setting row" do
+        expect(subject).not_to have_css(".govuk-summary-list__key", text: "Work setting")
+      end
+    end
+
+    context "when work_setting is blank" do
+      before do
+        application.update!(work_setting: "")
+        render partial: "accounts/work_details", locals: { application: application }
+      end
+
+      it "does not show the work setting row" do
+        expect(subject).not_to have_css(".govuk-summary-list__key", text: "Work setting")
+      end
+    end
+  end
+
+  describe "early years setting" do
+    context "when inside catchment and works in childcare" do
+      before do
+        allow(application).to receive(:inside_catchment?).and_return(true)
+        application.update!(works_in_childcare: true, kind_of_nursery: "private_nursery")
+        render partial: "accounts/work_details", locals: { application: application }
```

Comment:
```suggestion
        render partial: "accounts/work_details", locals: { application: }
```
and same for the others